docs/articles/expensify-classic/expensify-card/Request-the-Card.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Request the Card
+description: Details on requesting the Expensify Card as an employee
+---
+# Overview
+
+Once your organization is approved for the Expensify Card, you can request a card! 
+
+This article covers how to request, activate, and replace your physical and virtual Expensify Cards.
+
+# How to get your first Expensify Card
+
+An admin in your organization must first enable the Expensify Cards before you can receive a card. After that, an admin may assign you a card by setting a limit. You can think of setting a card limit as “unlocking” access to the card.
+
+If you haven’t been assigned a limit yet, look for the task on your account's homepage that says, “Ask your admin for the card!”. This task allows you to message your admin team to make that request.
+
+Once you’re assigned a card limit, we’ll notify you via email to let you know you can request a card. A link within the notification email will take you to your account’s homepage, where you can provide your shipping address for the physical card. Enter your address, and we’ll ship the card to arrive within 3-5 business days.
+ 
+Once your physical card arrives in the mail, activate it in Expensify by entering the last four digits of the card in the activation task on your account’s homepage. 
+
+# Virtual Card
+
+Once assigned a limit, a virtual card is available immediately. You can view the virtual card details via **Settings > Account > Credit Card Import > Show Details**. Feel free to begin transacting with the virtual card while your physical card is in transit – your virtual card and physical card share a limit.
+
+Please note that you must enable 2-factor authentication on your account if you want to have the option to dispute transactions made on your virtual card. 
```

Comment:
```suggestion
Please note that you must enable two-factor authentication on your account if you want to have the option to dispute transactions made on your virtual card. 
```
Numbers nine and below should be spelled out / this is consistent with the rest of the App and Web repos 😄

Reply:
ah, i love this. thank youuu!